### PR TITLE
Make all nodes schedulable by default in byo example inventory

### PR DESCRIPTION
**Pull request by sdodson**
... but note why you wouldn't want them to be

### CODE REVIEW

**Comment by sdodson** [comment]
@devop-mmcgrath I've added origin, atomic-enterprise, and openshift-enterprise examples here.

**Comment by detiber** [comment]
This is probably the best approach for the short term...  We can add followup items to genericize the variable names (and possibly the roles) at a later date, but we'll have to balance the importance of the naming with feature related changes.

File: inventory/byo/hosts.atomic-enterprise.example [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# defining an etcd group switches from embedded etcd to external etcd
+#[etcd]
+#aep3-etcd[1:3]-ansible.test.example.com
+
+# NOTE: Currently we require that masters be part of the SDN which requires that they also be nodes
+# However, in order to ensure that your masters are not burdened with running pods you should
+# make them unschedulable by adding openshift_scheduleable=False any node that's also a master.
+[nodes]
+aep3-master[1:3]-ansible.test.example.com
+aep3-node[1:2]-ansible.test.example.com openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```
Comment (sdodson) [inline inventory/byo/hosts.atomic-enterprise.example]: @devop-mmcgrath See above, do you want me to push for this to be merged as is or should I reset these hostnames back to ose3-* ? You'll also want to update the docs to reference this atomic-enterprise specific example hosts file. I've removed as many instances as 'openshift' as is safe to do without moving code around.

**Comment by devop-mmcgrath** [comment]
I hate to do this to you but can we wait until v3?  We've already had a lot of testing (and docs) adapted to ose3

**Comment by sdodson** [comment]
Dropped the example host file rename from the PR, this PR now only makes the masters schedulable by default.

**Comment by sdodson** [comment]
@brenton ready to merge

**Pull request merged**